docs/zones/zone-pivot-groups.yml

```diff
@@ -43,4 +43,13 @@ groups:
       title: Visual Studio Code
     - id: dotnet-cli
       title: .NET CLI
-
+- id: resp-host
+  title: RESP Host
+  prompt: Choose a RESP Hosting resource type
+  pivots:
+    - id: redis
+      title: Stack Exchange Redis
```

Comment:
This should be Redis, _Stack Exchange Redis_ refers to the client library which would be the same across all of the hosting resource types.